File: POO/Javascript/01-script.js
class Persona {
    constructor(nombre, edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    toString() {
        return `Hola, mi nombre es ${this.nombre} y tengo ${this.edad} años.`;
    }
}

const persona1 = new Persona("Alberto", 45);
console.log(persona1.toString());

File: POO/Java/01-script.java
public class Persona {
    private String nombre;
    private int edad;

    public Persona(String nombre, int edad) {
        this.nombre = nombre;
        this.edad = edad;
    }

    @Override
    public String toString() {
        return "Hola, mi nombre es " + nombre + " y tengo " + edad + " años.";
    }

    public static void main(String[] args) {
        Persona persona1 = new Persona("Alberto", 45);
        System.out.println(persona1);
    }
}